eng/pipelines/common/download-artifact-step.yml

```diff
@@ -19,6 +19,6 @@ steps:
   - task: ExtractFiles@1
     displayName: 'Unzip ${{ parameters.displayName }}'
     inputs:
-      archiveFilePatterns: $(Build.SourcesDirectory)/__download__/${{ parameters.artifactName }}/${{ parameters.artifactFileName }}
+      archiveFilePatterns: $(Build.SourcesDirectory)/__download__/${{ parameters.artifactName }}/**/${{ parameters.artifactFileName }}
```

Comment:
This one should be left as is. The uses always have `artifactName == artifactFileName`. That's why the problem only showed up in the internal installer. It only happens in the generation of . This little template is used in several places that expect only that specific output. Globbing should not be used here.